bolt/test/AArch64/r_aarch64_prelxx.s

```diff
@@ -21,7 +21,7 @@
 
 // 4 is offset in datatable
 // 8 is addend
-// CHECKPREL32: [[#DATATABLEADDR + 4 - 8 + VALUE]] <_start>:
+// CHECKPREL32: [[#DATATABLEADDR + VALUE]] <_start>:
```

Comment:
@yavtuk, I had to modify this test case as I believe it was broken. The relocations that are generated for `datatable` are pointing to phantom entry points in `_start`. As ADR in `_start` was previously relaxed into ADRP+ADD, the entry points were shifted by 4 bytes.